test/vstest.console.UnitTests/Internal/ConsoleLoggerTests.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.IO;
```

Comment:
nit:consider moving this into namespace block, after `using System;`

Reply:
done